readme/configuration-management.md

```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
 
 ## Configuration Split workflow
 
-BLT recommends using the Config Split module to manage configuration on most projects. For a description of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
+### Overview
+
+BLT recommends using the Config Split module to manage configuration on most projects. For an overview of how to use Config Split, see this excellent [blog post by Jeff Geerling](https://www.jeffgeerling.com/blog/2017/adding-configuration-split-drupal-site-using-blt-and-acquia-cloud).
 
 Note that this workflow currently has two major limitations. The first is that individual configurations can't be entirely excluded from configuration management. For instance, if you want administrators to be able to set the site name (as above) or create new contact forms / webforms in production, this would be difficult given the current state of the module. However, this should theoretically be possible.
 
 TODO: Update this documentation once Config Split's greylist functionality has been better documented and tested.
 
 The second limitation is that it's difficult to define configuration that varies between sites in a multisite installation. Multisite installations that require highly customized bundles of configuration per-site might be better suited by a Features-based workflow.
 
+### Setting up Config Split
+
+This section will guide you through initially setting up a Config Split workflow on your BLT based project.
+
+1. Ensure that you are using Drush version 8.1.10 or higher.
+2. Add Config Split version 8.x-1.0-beta4 or higher to your codebase: `composer require drupal/config_split`
+3. TODO: finish this section once the related functionality is stabilized in BLT core.
```

Comment:
We also need to indicate that `cm.strategy` must be set to `config-split` in project.yml. We may make this the default.